pathway-analysis/ora_with_webgestaltr.Rmd

```diff
@@ -80,7 +84,7 @@ We can identify upregulated genes by filtering these results and use this list
 as input to ORA.
 
 ```{r}
-dge_df <- readr::read_tsv("https://github.com/AlexsLemonade/refinebio-examples/raw/10b116dff0d44267db3e3a6fde735187e1bcd195/validate-differential-expression/results/GSE37418_limma_results.tsv")
+dge_df <- readr::read_tsv("https://github.com/AlexsLemonade/refinebio-examples/raw/10b116dff0d44267db3e3a6fde735187e1bcd195/validate-differential-expression/results/GSE37418_limma_results.tsv") # Replace with path or link to your desired differential expression results TSV file
```

Comment:
To make this clearer to the user, maybe declare this on the line before and call it something like `de_url` and then make it clear that it has to be a link to a TSV file OR a regular file path to a TSV one would have on their computer.

Reply:
Second question is do we want this to be a permalink to this particular file? or do we want to make it so its whatever the file is in the current release? Permalink is probably fine for now, but we could also revisit this later.